### ISSUE TITLE
The camera doesn't launch since SDK 45 only on iOS

### ISSUE THREAD

**Issue by TomCorvus** [expo/expo#17407]
### Summary

Hi 👋,
I updated Expo form 44 to 45 and all dependencies but since this update, the camera doesn't launch on iOS.
I'm using expo-camera to take photo.
`onCameraReady` and `onMountError` return nothing.

Everything works fine on Android.
Don't test on iOS simulators because it's working. Try only on physical devices.

Thanks 👍

### Managed or bare workflow? If you have `ios/` or `android/` directories in your project, the answer is bare!

bare

### What platform(s) does this occur on?

iOS

### Environment

```
  expo-env-info 1.0.3 environment info:
    System:
      OS: macOS 12.3.1
      Shell: 5.8 - /bin/zsh
    Binaries:
      Node: 14.17.6 - /usr/local/bin/node
      Yarn: 3.2.0 - /usr/local/bin/yarn
      npm: 8.9.0 - /usr/local/bin/npm
      Watchman: 2022.03.21.00 - /usr/local/bin/watchman
    Managers:
      CocoaPods: 1.11.3 - /usr/local/bin/pod
    SDKs:
      iOS SDK:
        Platforms: DriverKit 21.4, iOS 15.4, macOS 12.3, tvOS 15.4, watchOS 8.5
    IDEs:
      Android Studio: 2021.1 AI-211.7628.21.2111.8193401
      Xcode: 13.3.1/13E500a - /usr/bin/xcodebuild
    npmPackages:
      expo: ^45.0.1 => 45.0.1 
      react: ^17.0.2 => 17.0.2 
      react-native: ~0.68.1 => 0.68.1 
      react-native-web: ^0.17.7 => 0.17.7 
    npmGlobalPackages:
      eas-cli: 0.8.1
      expo-cli: 5.4.3
    Expo Workflow: bare
```

### Reproducible demo

https://github.com/TomCorvus/expo-camera-not-working

I can't create a snack with Expo SDK 45 because it is not available.

**Comment by fobos531** [comment]
Related and possible duplicate of #17395

**Comment by hirbod** [comment]
Its still good to have 2 isolated tickets, one for video, one for camera

**Comment by TomCorvus** [comment]
Yes, I saw this issue, but as herbed said, I think it is not related. I'm surprised no one has already noticed this issue. Maybe I forgot something during the update.

**Comment by codybrouwers** [comment]
Am also seeing this issue, downgrading the version of `expo-camera` to 12.1.2 didn't fix the issue. `onCameraReady` is still never called and the preview is not visible.
